docs/dice_verification.md

```diff
@@ -1,40 +1,137 @@
 ## Verifying dice seed generation 
 
 It is possible to do a 'dry run' to verify that seed generation has not been tempered.
-This will ensure that the derivation algorithm has not been tempered and is still same as well known algorithm used bu coldcard and ian coleman bip39 webpage.
+This will ensure that the derivation algorithm has not been tempered and is still the same as the well known algorithm used by coldcard and Ian Coleman bip39 webpage.
```

Comment:
Please correct the spelling of tempered to tampered. its incorrect in quite a few places.